src/com/sds/DBHandler.java

```diff
@@ -37,16 +41,81 @@
 	public DBHandler() throws SQLException {
 		this.con = DriverManager.getConnection(URL,USER,PASS);
 	}
-	String[][] contractArray = new String[3][3];
+	
+	public void searchForContract() 
+	{
+		Scanner scan = new Scanner(System.in);
+		System.out.println("Enter contract name to search for:");
+		String searchByName = scan.nextLine();
+		try
+		{
+			searchByName = SEARCH_CONTRACT + "'" + searchByName + "'";

```

Comment:
this works fine, but you could also use the prepared statement with a '?' instead. (see the get files method). No changes needed but I thought i'd point it out